BGAnimations/ScreenSelectMusic overlay/PaneDisplay.lua

```diff
@@ -106,13 +115,21 @@ local GetScoresRequestProcessor = function(res, master)
 					end
 
 					if gsEntry["isSelf"] then
-						SetNameAndScore(
-							GetMachineTag(gsEntry),
-							string.format("%.2f%%", gsEntry["score"]/100),
-							playerName,
-							playerScore
-						)
-						personalRecordSet = true
+						-- Let's check if the GS high score is higher than the local high score
+						local player = PlayerNumber[i]
+						local localScore, localName = GetLocalScoreAndNameForPlayer(player)
+						local gsScore = gsEntry["score"]
+						localScore = localScore and tonumber(localScore:gsub("%%", "") * 100) or nil
+						if not localScore or gsScore > localScore then
```

Comment:
I think we should also prefer the GS score if it's the same as the localScore, so that the name we display is consistent with the user's machine tag on GS. For my local setup I would otherwise see EVNT as the name for all scores.

Reply:
Ah good idea I agree!

Reply:
Oh yeah, that would make a lot of sense! I hadn't noticed since my local scores use the same tag :sweat_smile: